### PR TITLE
Help: fix crash

### PR DESCRIPTION
**Pull request by mia-pi-git**

### CODE REVIEW

File: server/chat-plugins/help.ts [hunk 1]
```diff
@@ -69,7 +67,7 @@ export class HelpResponder {
 	queue: string[];
 	data: PluginData;
 	constructor(data: PluginData) {
-		this.data = data;
+		this.data = data || {};
```
Comment (Zarel) [inline server/chat-plugins/help.ts:70]: It does not seem like `|| {}` should be happening at all.

Comment (mia-pi-git) [reply server/chat-plugins/help.ts]: No, that should be throwing at this point, you're right.

File: server/chat-plugins/help.ts [hunk 2]
```diff
@@ -15,18 +15,16 @@ const PATH = 'config/chat-plugins/help.json';
...
-} catch (e) {
-	helpData = {
-		stats: {},
-		pairs: {},
-		disabled: false,
-		queue: [],
-	};
-}
+	helpData = JSON.parse(FS(PATH).readIfExistsSync());
+} catch (e) {}
```
Comment (Zarel) [inline server/chat-plugins/help.ts:27]: I'm not sure the catch is correct, either. The only error we want to suppress is "doesn't exist". We should actually just use a regular `read`, and re-throw if we get any error other than "file not found".

Comment (Zarel) [reply server/chat-plugins/help.ts:27]: We can even initialize to your initial state there.

**Comment by mia-pi-git** [comment]
(npm fulltest is passing fine on my own machine, fwiw)

**Pull request merged**